includes/Core/Conversion_Tracking/Conversion_Tracking.php

```diff
@@ -146,9 +146,13 @@ function ( Conversion_Events_Provider $active_provider ) {
 				}, 5 );
 
 				gtag( "event", name, { ...data, event_source: "site-kit" } );
-			}
+			};
 		';
 
+		if ( function_exists( 'edd_get_currency' ) ) {
+			$gtag_event .= "window._googlesitekit.eddCurrency = '" . edd_get_currency() . "';";
```

Comment:
Let's be more explicit with this variable name:

```suggestion
			$gtag_event .= "window._googlesitekit.easyDigitalDownloadsCurrency = '" . edd_get_currency() . "';";
```